app/src/main/java/at/roteskreuz/stopcorona/model/entities/infection/exposure_keys/IndexOfDiagnosisKeysArchives.kt

```diff
@@ -2,24 +2,42 @@ package at.roteskreuz.stopcorona.model.entities.infection.exposure_keys
 
 import com.squareup.moshi.Json
 import com.squareup.moshi.JsonClass
+import org.threeten.bp.ZonedDateTime
+import java.time.DateTimeException
 
 /**
  * Describes infection info about user with data gathered from the Exposure SDK.
  */
 @JsonClass(generateAdapter = true)
 data class IndexOfDiagnosisKeysArchives(
-    @field:Json(name = "full_batch")
-    val fullBatch: DiagnosisKeysBatch,
+    @field:Json(name = "full_14_batch")
+    val full14DaysBatch: DiagnosisKeysBatch,
```

Comment:
Please check again your nullability in api entities. Is it correct? Safer is to use nullable fields everywhere and log error in case of null.

Reply:
@falkorichter

Reply:
Please add new ApiError which will be explicitely handled in some try-catch block.
In case of nullable parse failed.